### ISSUE TITLE
cooler and pairsamtools compatibility issue

### ISSUE THREAD

**Issue by nandankita** [mirnylab/cooler#115]
Was the compatibility issue between `cooler` and `pairsamtools` fixed?
https://github.com/mirnylab/distiller-nf/issues/89

P.S.: `pairsamtools` has 1 based indexing for pairs (SAM style)
`cooler` used to use 0 based indexing for pairs, did it change?

**Comment by nandankita** [comment]
Thanks for the fix! I am running the distiller now. Will update once it's done.

**Comment by nandankita** [comment]
It failed giving the following error. 
```
INFO:cooler:Binning chr41:0-9688883|*
  Traceback (most recent call last):
    File "/home/an27w/lib/miniconda3/bin/cooler", line 11, in <module>
      load_entry_point('cooler==0.7.9.dev0', 'console_scripts', 'cooler')()
    File "/home/an27w/lib/miniconda3/lib/python3.6/site-packages/click/core.py", line 722, in __call__
      return self.main(*args, **kwargs)
    File "/home/an27w/lib/miniconda3/lib/python3.6/site-packages/click/core.py", line 697, in main
      rv = self.invoke(ctx)
    File "/home/an27w/lib/miniconda3/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
      return _process_result(sub_ctx.command.invoke(sub_ctx))
    File "/home/an27w/lib/miniconda3/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
      return _process_result(sub_ctx.command.invoke(sub_ctx))
    File "/home/an27w/lib/miniconda3/lib/python3.6/site-packages/click/core.py", line 895, in invoke
      return ctx.invoke(self.callback, **ctx.params)
    File "/home/an27w/lib/miniconda3/lib/python3.6/site-packages/click/core.py", line 535, in invoke
      return callback(*args, **kwargs)
    File "/home/an27w/lib/miniconda3/lib/python3.6/site-packages/cooler/cli/cload.py", line 261, in pairix
      create(cool_path, bins, iterator, metadata, assembly)
    File "/home/an27w/lib/miniconda3/lib/python3.6/site-packages/cooler/io.py", line 286, in create
      file_path, target, meta.columns, iterable, h5opts, lock)
    File "/home/an27w/lib/miniconda3/lib/python3.6/site-packages/cooler/_writer.py", line 203, in write_pixels
      for i, chunk in enumerate(iterable):
    File "/home/an27w/lib/miniconda3/lib/python3.6/site-packages/cooler/_binning.py", line 1350, in _validate_pixels
      "Found a bin ID that exceeds the declared number of bins. " 
  cooler._binning.BadInputError: Found a bin ID that exceeds the declared number of bins. Check whether your bin table is correct.
```

**Comment by nvictus** [comment]
Fixed in release v0.7.9.
